pkg/util/file.go

```diff
@@ -5,6 +5,14 @@ import (
 	"io"
 	"io/fs"
 	"os"
+	"syscall"
+)
+
+const (
+	FileMode600 = fs.FileMode(0600) // User: read/write, Group: none, Others: none
+	FileMode640 = fs.FileMode(0640) // User: read/write, Group: read-only, Others: none
+	FileMode700 = fs.FileMode(0700) // User: full access, Group: none, Others: none
+	FileMode750 = fs.FileMode(0750) // User: full access, Group: read/execute only, Others: none
```

Comment:
I feel like it'd be nicer if we name these constants after their permissions, something like:
```suggestion
	FileModeUserReadWrite      = fs.FileMode(0600) // User: read/write, Group: none, Others: none
	FileModeUserGroupReadWrite = fs.FileMode(0640) // User: read/write, Group: read-only, Others: none
	FileModeUserFull           = fs.FileMode(0700) // User: full access, Group: none, Others: none
	FileModeUserFullGroupRead  = fs.FileMode(0750) // User: full access, Group: read/execute only, Others: none
```

Though, feel free to change them if you can come up with better names :)

Reply:
Will change in the next revision. Do not have better naming suggestion.